.pre-commit-config.yaml

```diff
@@ -6,18 +6,18 @@ repos:
     - id: go-vet
     - id: go-lint
  - repo: https://github.com/keep-network/pre-commit-hooks.git
-   rev: 63e729f
+   rev: v1.3.0
    hooks:
     - id: check-added-large-files
  - repo: local
    hooks:
-    - id: solidity-lint-js
+    - id: lint-js
       name: 'solidity lint js'
       entry: /usr/bin/env bash -c "cd solidity && npm run lint:js"
-      files: '\.js$'
+      files: 'solidity\/.*\.js$'
       language: script
       description: "Checks JS code according to the package's linter configuration"
-    - id: solidity-lint-sol
+    - id: lint-sol
```

Comment:
Let's leave the `solidity-` prefix as we're still in the solidity directory.

Reply:
Done